src/js/store/perfdata/selectors.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import type { AppState } from 'store';
+import type { PerfDataState, LoadingStatus } from 'store/perfdata/reducer';
+import type {
+  TestMethodPerfUI,
+  FilterDefinition,
+} from 'api/testmethod_perf_UI_JSON_schema';
+
+export const selectPerfState = (appState: AppState): PerfDataState =>
+  appState.perfData;
+
+export const selectPerfUIStatus = (appState: AppState): LoadingStatus =>
+  appState.perfDataUI ? appState.perfDataUI.status : 'LOADING';
+
+export const selectPerfDataAPIUrl = (appState: AppState): string =>
+  appState.perfData.status === 'AVAILABLE' ? appState.perfData.url : '';
```

Comment:
When selecting nested state, I would use reselect's `createSelector` helper to allow one selector to rely on another. See https://github.com/SFDO-Tooling/MetaDeploy/blob/master/src/js/store/products/selectors.js#L58-L72 for a simple example.